src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+import cn from 'classnames';
+interface Props {
+  uncompletedTodosLength: number;
+  filterBy: FilterTodosBy;
+  setFilteredBy: (filterBy: FilterTodosBy) => void;
+  completedTodosLenght: number;
+  handleDeleteAllCompleted: () => void;
+}
+export const Footer: React.FC<Props> = props => {
+  const {
+    uncompletedTodosLength,
+    filterBy,
+    setFilteredBy,
+    completedTodosLenght,
+    handleDeleteAllCompleted,
+  } = props;
+
+  const filters = Object.values(FilterTodosBy);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {uncompletedTodosLength} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {filters.map(filter => (
+          <a
+            key={filter}
+            href={`/#/${filter === FilterTodosBy.All ? '' : filter.toLowerCase()}`}
+            className={cn('filter__link', {
+              selected: filterBy === filter,
+            })}
+            data-cy={`FilterLink${filter}`}
+            onClick={() => setFilteredBy(filter)}
+          >
+            {filter}
+          </a>
+        ))}
+      </nav>
+
+      <button
+        type="button"
+        className="todoapp__clear-completed"
+        data-cy="ClearCompletedButton"
+        disabled={completedTodosLenght === 0}
```

Comment:
will be the same(better option). It will be converted to boolean value
```suggestion
        disabled={!completedTodosLenght}
```